src/main/resources/assets/computercraft/lua/rom/apis/colors

```diff
@@ -25,13 +25,23 @@ function combine( ... )
 end
 
 function subtract( colors, ... )
-	local r = colors
-	for n,c in ipairs( { ... } ) do
-		r = bit32.band(r, bit32.bnot(c))
-	end
-	return r
+    local r = colors
+    for n,c in ipairs( { ... } ) do
+        r = bit32.band(r, bit32.bnot(c))
+    end
+    return r
 end
 
 function test( colors, color )
     return ((bit32.band(colors, color)) == color)
 end
+
+function rgb8( r, g, b )
```

Comment:
Surely the more useful things to do are:
colours.rgb8(0x00ffff) -> 0.0, 1.0, 1.0
colours.rgb8(0.0, 1.0, 1.0) -> 0x00ffff